language/nodejs/develop.md

```diff
@@ -140,7 +140,7 @@ volumes:
 
 This Compose file is super convenient as we do not have to type all the parameters to pass to the `docker run` command. We can declaratively do that in the Compose file.
 
-We are exposing `port 9229` so that we can attach a debugger. We are also mapping our local source code into the running container so that we can make changes in our text editor and have those changes picked up in the container.
+We are exposing `port 9229` so that we can attach a debugger. The “volumes” means that we are also mapping our local source code into the running container so that we can make changes in our text editor and have those changes picked up in the container.
```

Comment:
```suggestion
We are exposing `port 9229` so that we can attach a debugger. With `volumes`,  we are also mapping our local source code into the running container so that we can make changes in our text editor and have those changes picked up in the container.
```